contrib/depends/packages/boost.mk

```
@@ -1,6 +1,6 @@
 package=boost
 $(package)_version=1_73_0
-$(package)_download_path=https://downloads.sourceforge.net/project/boost/boost/1.73.0/
+$(package)_download_path=https://gitlab.com/ArqTras/depends/-/raw/main/
```

why not official boost link ?
```suggestion
$(package)_download_path=https://sourceforge.net/projects/boost/files/boost/1.73.0/
```

---

Because sometimes sourceforge is not available.Reason to move it in to gitlab is to not search each time for mirror when they turn off repo like with downloads.bitcoin

---

@up